packages/coinstac-ui/app/render/components/notification.js

```diff
@@ -18,6 +30,19 @@ export default class Notify extends React.Component {
     app.notify = (level, message) => this.push({ level, message, autoDismiss: 2 });
   }
 
+
+  componentWillUnmount() {
+    ipcRenderer.removeListener('async-error', this.handleAsyncError);
+  }
+
+  handleAsyncError(event, message) {
+    this.push({
+      autoDismiss: 0,
+      level: 'error',
+      message: truncate(message, { length: 250 }),
+    });
+  }
```

Comment:
It is here: https://github.com/MRN-Code/coinstac/pull/144/files#diff-e68a4ebae3f693453666abc796d7a76cR17
:-)